frontend/public/components/utils/headings.tsx

```diff
@@ -54,11 +58,22 @@ export const PageHeading = connectToModel((props: PageHeadingProps) => {
   const hasButtonActions = !_.isEmpty(buttonActions);
   const hasMenuActions = _.isFunction(menuActions) || !_.isEmpty(menuActions);
   const showActions = (hasButtonActions || hasMenuActions) && !_.isEmpty(data) && !_.get(data, 'deletionTimestamp');
-
   return <div className={classNames('co-m-nav-title', {'co-m-nav-title--detail': detail}, {'co-m-nav-title--logo': isCSV}, {'co-m-nav-title--breadcrumbs': breadcrumbsFor && !_.isEmpty(data)})} style={style}>
-    { breadcrumbsFor && !_.isEmpty(data) && <BreadCrumbs breadcrumbs={breadcrumbsFor(data)} /> }
+    <Split style={{alignItems: 'baseline'}}>
```

Comment:
Only add this when `breadcumbsFor` is truthy.

Reply:
Done